src/contracts/libraries/GPv2Encoding.sol

```diff
@@ -83,36 +83,40 @@ library GPv2Encoding {
         hex"b2b38b9dcbdeb41f7ad71dea9aed79fb47f7bbc3436576fe994b43d5b16ecdec";
 
     /// @dev The stride of the fixed-length components in an encoded trade.
-    uint256 private constant FIXED_LENGTH_TRADE_STRIDE = 206;
+    uint256 private constant FIXED_LENGTH_TRADE_STRIDE = 141;
+
+    /// @dev The stride of any signature from an externally owned account.
+    uint256 private constant EOA_SIGNATURE_STRIDE = 65;
```

Comment:
naming-nit: I woulnd't consider these "strides" as they aren't offsets between successive `Trade`s and `Signature`s in an array. I think these should be:
```suggestion
    /// @dev The size of the fixed-length components in an encoded trade.
    uint256 private constant FIXED_LENGTH_TRADE_STRIDE = 141;

    /// @dev The byte length of ay signature from an externally owned account.
    uint256 private constant EOA_SIGNATURE_STRIDE = 65;
```

Reply:
Fixed as:
```
    /// @dev The length of the fixed-length components in an encoded trade.
    uint256 private constant CONSTANT_SIZE_TRADE_LENGTH = 141;

    /// @dev The length of any signature from an externally owned account.
    uint256 private constant ECDSA_SIGNATURE_LENGTH = 65;
```